buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -1044,15 +1044,7 @@ def create_job(JOB_NAME, SDK_VERSION, SPEC, downstreamJobType, id){
     params.put('DISCARDER_NUM_BUILDS', DISCARDER_NUM_BUILDS)
 
     def templatePath = 'buildenv/jenkins/jobs/pipelines/Pipeline_Template.groovy'
-    ret = false    
-    retry(3) {
-       if (ret) {
-          sleep time: 120,unit: 'SECONDS'
-       } else {
-          ret = true
-       }
-       jobDsl targets: templatePath, ignoreExisting: false, additionalParameters: params
-    }
+    retry_and_delay({jobDsl targets: templatePath, ignoreExisting: false, additionalParameters: params}, 3, 120)
```

Comment:
You'll have to check if `pipeline-functions.groovy` is loaded at this point in the build and what variable it's loaded into. Same comment for build.groovy. Look for a `load` step
ex `buildFile = load 'buildenv/jenkins/common/pipeline-functions.groovy'`

Reply:
Hey Adam! I think ```pipeline-functions.groovy``` is not loaded. I checked the console output and there is no line with load. Thanks!

Reply:
Right, so we'll have to make sure we load that library in those files so we can utilize this new function.